### ISSUE TITLE
Votes are not longer on blockchain, but marked as voted in UI

### ISSUE THREAD

**Issue by punkrock** [LiskHQ/lisk#73]
Before we had the trouble after starting 0.2.0 yesterday, I voted for some delegates. After the issue was fixed and the blocks kept on rollin', my votes didn't longer show up in the "My Votes"-tab. I asked some delegates, if they can verify that I voted for them. They denied. So the votes are not longer on the blockchain. The problem is: The votes are still (15 hours later) marked as voted in my "Delegates"-tab. So, I am not able to vote them anymore.

Reference:
` {
      "fee": 100000000, 
      "timestamp": 33419949, 
      "blockId": "10878028191552750370", 
      "senderId": "16671600588889208369L", 
      "recipientId": "16671600588889208369L", 
      "senderPublicKey": "3193057832bb1c9782a8e4a32e543b535ed9d750b1b10383f8b6f50853569609", 
      "height": 316, 
      "amount": 0, 
      "asset": {
        "votes": [
          "+18d1b236b98b02c464bbe057077c7dea32b4f519f573b4b2de57ea0634e64a7c", 
          "+2068bdbac2f0545fd3dbedb7245d7d2895687a6c3066f492c47e3de2cc93d122", 
          "+220305e5ecb974cd8d5931cf31653fd7a01fdcb6862a49d693c035807932896f", 
          "+263b86aaa8a9fa3a36648895178fcf911c91b3fccb188a2ddf013acefc674989", 
          "+961d1a1057a09f865291873e9ba3d0af7b2a3a1e971bb7576a2aab1c526acbcd", 
          "+bff6075b423e9d592178b7d838ceef8062debf1ba239a7fe66704c46d369ace3", 
          "+e76da08089a365d35384133858cb0b4f4eae63f0993f3868f2ca6ae087ed32fa", 
          "+dc979116a6513fd00a0569d9f2f7f4776432328584494100998e594c2bbea79a", 
          "+df16fd66d063e5652e4a5bf3b949119bc469aa46a61f7199fa93c08013f32c29", 
          "+3193057832bb1c9782a8e4a32e543b535ed9d750b1b10383f8b6f50853569609", 
          "+8f730aae6715d2b85584246a1b807c6092f0abb4eefd695b26b26c9573f00537", 
          "+4c6a450cc6769efa4ba0f9a23318af0cb9def2402f0a51c5e7215856c08df7af", 
          "+f2f4b102b0889b32bbc7cb0767bfa4f740a9ae1dfc7d981380778b97fb42acb7", 
          "+cbf4ed7dbc6054b70e3744ce0150be4151e2cd99955cbffa19e3158b91739652", 
          "+e2b26b11793b8be3d7a8e65dc5dde50e6e9b644270e3d77aa742e971fd83b301", 
          "+abf9787621f8f43ec4e4a645b515094f42fc5615f2e231eca24eaf6e69dc6a65", 
          "+b333ac456897b92acdc6e2b9c5213ac7175360dd25aff3899198e526dc5aeb27", 
          "+b6de69ebd1ba0bfe2d37ea6733c64b7e3eb262bee6c9cee05034b0b4465e2678", 
          "+29defafb54e24b7e5a4c08c4fb14d8f236d1e1f5856c843413ad7b5dc52d6975", 
          "+19115966193c80dbf0ab0cdf3c53f3b105b846ab58ac6255e485d8e5f6da79ce", 
          "+0348a623c41ed7742a7f35a5812476750e2ba41208e16a29b110e6fe11e514d4", 
          "+60c0f74e510a81d6dbaa7b93e5237fedbd8fe44c21ddfd8eb76c2ae1a994b9a2", 
          "+90ad9bfed339af2d6b4b3b7f7cdf25d927b255f9f25dbbc892ee9ca57ef67807", 
          "+70ce0c2e6ff3f81196ffab7c3e73e08f55abf13c484e48d663d06813dc7932e8", 
          "+cc1c5f58cd761ef677310029bc2f03058905afacf3ef4b7c67360141f2ae1208", 
          "+d6a5590caba0bd80eaaaee985e0e33a87b082d9b65d2ee4d2e21845625630969", 
          "+81df22a74fc8e87a1699d2fd0651cdb2f704647d5f7c3b43888c8b54848f7d32", 
          "+27c7ed92b0edac6387b65cc95ce6c2b8e5efc7827fd5b5c7995e6356b276ae93", 
          "+2a1a35e14f35daa86c4f1a566566b54b39321ac97c33ab0bb30a0b56764e0920"
        ]
      }, 
      "signature": "b11ecdb2b0cf71ffaa975b432bbb164cdfe5a421f2ba305397bfb20c3caba5cf731f3a862638020173976177ad5c61bc5ed21dcd0b8515520b4395a29bdb7a04", 
      "type": 3, 
      "id": "7991225248478826612"
    }`

**tl;dr:** I voted, but votes are gone from the blockchain, but still marked in the UI at the "Delegates"-tab as voted.

**Comment by fix** [comment]
I think this is more a lisk-ui bug. The thing is that angular does not always refresh the UI as expected. If you navigate in the interface and get back to the vote page, it usually fixes the issue.

**Comment by fix** [comment]
closing as duplicate of https://github.com/LiskHQ/lisk-ui/issues/39
